engines/director/detection_tables.h

```diff
@@ -527,6 +527,34 @@ static const DirectorGameDescription gameDescriptions[] = {
 		2
 	},
 
+	{
+		{
+			"ernie",
+			"Ernie - Demo",
```

Comment:
Since you're adding another target, you need to register it in the detection.cpp file as well.